tests/core/test_agent.py

```diff
@@ -74,6 +74,7 @@ async def test_training_data_is_reproducible():
         assert str(x.as_dialogue()) == str(same_training_data[i].as_dialogue())
 
 
+@pytest.mark.trains_model
```

Comment:
what about `test_load_agent`?

Reply:
Are you suggesting renaming the marker or the test?

Reply:
adding a marker

Reply:
haha, seems resolved 👍🏻